todo_app/data/trello_items.py

```diff
@@ -41,23 +41,42 @@ def add_item(title):
     params=query
   )
 
+def _get_status_from_list_id(list):
+  if list == os.getenv("TRELLO_FINISHED_LIST_ID"):
+    return "Completed"
+  if list == os.getenv("TRELLO_IN_PROGRESS_LIST"):
+    return "In Progress"
+  if list == os.getenv("TRELLO_NOT_STARTED_LIST_ID"):
+    return "Not Started"
+
 def get_items():
-  url = f"{get_api_url()}/boards/{os.getenv("TRELLO_BOARD_ID")}/cards"
-  response = requests.request(
-    "GET",
+  id = os.getenv("TRELLO_BOARD_ID")
+  url = f"{get_api_url()}/boards/{id}/cards"
+  response = requests.get(
     url,
     headers=get_headers(),
     params=get_base_query()
   )
   items = []
-  body = json.loads(response.text)
+  body = response.json()
   for card in body:
-    items.append(Item(card["id"], card["name"], "Completed" if card["idList"] == os.getenv("TRELLO_FINISHED_LIST_ID") else "Not Started"))
+    items.append(Item(
+      card["id"],
+      card["name"], 
+      _get_status_from_list_id(card["idList"])))
```

Comment:
Really minor, but would it be nicer to have this fire as the default approach in the `from_trello_card` method? In particular since the logic in there currently continues to assume only two lists, which isn't necessarily our assumption now